django/contrib/contenttypes/utils.py

```diff
@@ -0,0 +1,8 @@
+from django.apps import apps
```

Comment:
Thanks for the feedback. However, "remove" is consistent with it's use throughout the management.py file and the reason for the choice of wording (`force_remove`, `remove_contenttypes`, `to_remove`, etc). Why do we want to break this consistency with "prune"?